solr/core/src/java/org/apache/solr/schema/DenseVectorField.java

```diff
@@ -142,8 +134,8 @@ public VectorSimilarityFunction getSimilarityFunction() {
     return similarityFunction;
   }
 
-  public String getCodecFormat() {
-    return codecFormat;
+  public String getAlgorithm() {
+    return algorithm;
```

Comment:
With backwards compatibility in mind, could `codecFormat` perhaps be deprecated before it's removed e.g.

Solr 9.1
* `knnAlgorithm="..."` is supported
* `codecFormat="..."` is deprecated but still supported
* combination of `codecFormat="..."` and `knnAlgorithm="..."` is either not supported (throws error) or the `codecFormat` is logged as a warning and then ignored. whichever choice is easier to implement?

Solr 9.2 or higher
* only `knnAlgorithm="..."` is supported

Reply:
Hi @cpoerschke ,
I thought I responded here as well but I managed to forget...
So, thanks for the suggestion, and  in certain scenarios I would agree but being an "experimental" feature I think we can just keep it simple and do that here in this pull request.
In this way, we'll avoid the complexity of having to create another pull request immediately after (and it's a matter of 1 version or another anyway)